### DISCUSSION TITLE
quot & 'sym

### DISCUSSION THREAD

**Discussion by ghost** [h3rald/min#145] [Q&A]
I didn't understand the differencies of these (quot & 'sym). What are the differencies of them?

**Comment by h3rald** [comment]
Well...

* in case of `a quote`:
    * evaluates a
    * evaluates quot (so quotes *the value* of a)
* in case of `'a`:
    * evaluates "a"
    * evaluates ' --> quote
    * err ok, you win, 😬 I just checked that in this case it creates a symbol using "a"

Theoretically, `'a` should be equal to  `("a")` actually, but it would also be pretty useless. Instead, `'` is extremely important in min, in its current implementation!

I should however clarify this. The easiest way to fix this inconsistency would be to create a new symbol `quotesym` that takes a string and converts it into a quoted symbol, and then say that the `'` sigil is equivalent to that... 😜

Unfortunately there's also a symbol `'` (not sigil!) which is also currently the equivalent for `quote`. 🙄

**Reply by ghost** [reply]
> The easiest way to fix this inconsistency would be to create a new symbol `quotesym`

Okay, looks fine. This may does better.

> Unfortunately there's also a symbol ' (not sigil!) 

Looks like i didn't understand again. It must be a sigil. Isn't `'` a sigil?
The min website says also that is a sigil.

You may give different symbols for them instead of giving `'` to all them.

**Reply by h3rald** [reply]
There's a ' sigil and a ' symbol... but the problem right now is that the sigil behaves a little differently, as you noticed.

From next version, ' will be an alias for the new quotesym symbol and a sigil that will behave in the same way.
